app/assets/javascripts/login.js

```diff
@@ -46,4 +46,16 @@ $(document).ready(function() {
       $(".login-flashes").append("<p class='text-danger'>Please verify it is correct and try again</p>");
```

Comment:
For all of the text-danger and text-successes, could we add a role='alert' on these too just for accessibility stuff for the workflow of resetting password for example from the modal, not from the reset password page.

ex.
```js
$(".login-flashes").append("<p class='text-danger'>Invalid e-mail address</p>");
```
turn into:
```js
$(".login-flashes").append("<p class='text-danger' role='alert'>Invalid e-mail address</p>");
```

Reply:
We should loop that in under another ticket since that is related to the other login/change password workflow from the pop-up dialog (versus from the /users/sign_in URL)